openedx/core/djangoapps/oauth_dispatch/views.py

```diff
@@ -138,19 +125,6 @@ class AccessTokenExchangeView(_DispatchingView):
     """
     dot_view = auth_exchange_views.DOTAccessTokenExchangeView
 
-    def get_view_for_backend(self, backend):
-        """
-        Return the appropriate view from the requested backend.
-        Since AccessTokenExchangeView no longer supports dop, this function needed to
-        be overwritten from _DispatchingView, it was decided that the dop path should not be removed
-        from _DispatchingView due to it still being used in other views(AuthorizationView, AccessTokenView)
```

Comment:
I'm not 100% clear on what was meant by this comment and why this can go away, although it is probably fine.  Can you add context?

Reply:
this was added in https://github.com/edx/edx-platform/pull/23187 as an interm solution. This function in AccessTokenExchangeView(_DispatchingView) overwrote a function in _DispatchingView which required DOP. Overwriting the function allowed us to remove DOP from AccessTokenExchangeView without it requiring us to remove DOP everywhere.

Reply:
this was added in https://github.com/edx/edx-platform/pull/23187 as an interm solution. This function in AccessTokenExchangeView(_DispatchingView) overwrote a function in _DispatchingView which required DOP. Overwriting the function allowed us to remove DOP from AccessTokenExchangeView without it requiring us to remove DOP everywhere.

Reply:
this was added in https://github.com/edx/edx-platform/pull/23187 as an interm solution. This function in AccessTokenExchangeView(_DispatchingView) overwrote a function in _DispatchingView which required DOP. Overwriting the function allowed us to remove DOP from AccessTokenExchangeView without it requiring us to remove DOP everywhere.